src/shared/services/case-fields/format-translator.service.ts

```diff
@@ -169,4 +174,54 @@ export class FormatTranslatorService {
     }
     return result.join('');
   }
+
+  removeTime(dateFormat: string): string {
+    while (dateFormat.includes('H') || dateFormat.includes('h')) {
+      dateFormat = dateFormat.replace('H', '');
+      dateFormat = dateFormat.replace('h', '');
+    }
+    while (dateFormat.includes('m')) {
+      dateFormat = dateFormat.replace('m', '');
+    }
+    while (dateFormat.includes('S') || dateFormat.includes('s')) {
+      dateFormat = dateFormat.replace('S', '');
+      dateFormat = dateFormat.replace('s', '');
+    }
+    return dateFormat;
```

Comment:
Yeah, as mentioned in the test, this doesn't seem to tidy up after itself when removing time. In the test example, it leaves ` ::` for a full `HH:mm:ss` timestamp. It'd leave ` :` for just `HH:mm` and it doesn't look like it handles milliseconds at all... so `HH:mm:ss.fff` (`10:19:28.123`) would leave you with ` ::.fff` (` ::.123`), which is clearly not what you want.

Reply:
Milliseconds are determined by character 'S' for datetime picker so they would be removed. Again, could make assumptions on formatting? Remove full stops and colons when removing time?

Reply:
Changes made as per above